src/helper.tsx

```diff
@@ -0,0 +1,55 @@
+import { Person, SortBy } from './types';
+
+export const getFilteredPeople = (
+  people: Person[],
+  sortField: keyof Person,
+  sortOrder: string,
+  filterSex: string,
+  searchQuery: string,
+  filterByCentries: string[],
+) => {
+  let preparedPeople = [...people];
+
+  if (filterByCentries.length) {
+    preparedPeople = preparedPeople.filter(person => {
+      const mathCentry = Math.ceil(person.born / 100).toString();
+
+      return filterByCentries.includes(mathCentry);
+    });
+  }
+
+  if (searchQuery) {
+    preparedPeople = preparedPeople.filter(person => (
+      person.name.toLowerCase().includes(searchQuery.toLocaleLowerCase().trim())
```

Comment:
split it with variables
Also, DRY, so create a separate function to format & check